src/internal/components/button-trigger/styles.scss

```diff
@@ -29,8 +29,8 @@ $padding-block-inner-filtering-token: 0px;
   border-start-end-radius: styles.$control-border-radius;
   border-end-start-radius: styles.$control-border-radius;
   border-end-end-radius: styles.$control-border-radius;
-  border-block: styles.$control-border-width solid awsui.$color-border-input-default;
-  border-inline: styles.$control-border-width solid awsui.$color-border-input-default;
+  border-block: awsui.$border-width-field solid awsui.$color-border-input-default;
+  border-inline: awsui.$border-width-field solid awsui.$color-border-input-default;
```

Comment:
Should the background of these buttons be white or grey
https://d21d5uik3ws71m.cloudfront.net/components/565543accaf89e219bb37bcfc45e2946528ffeb8/dev-pages/index.html#/light/app-layout/with-table-collection-select-filter/

<img width="332" alt="image" src="https://github.com/user-attachments/assets/beb8f09d-54cf-4b87-ad13-a370aada7b43" />

Reply:
They should remain white